src/app/store/actions/VehicleTestResultModel.actions.spec.ts

```diff
@@ -1,60 +1,88 @@
 import {
-  EVehicleTestResultModelActions, GetVehicleTestResultModel, GetVehicleTestResultModelSuccess, GetVehicleTestResultModelFailure
+  EVehicleTestResultModelActions,
+  GetVehicleTestResultModel,
+  GetVehicleTestResultModelFailure,
+  GetVehicleTestResultModelSuccess,
+  SetCurrentState,
+  SetSelectedTestResultModel,
+  SetSelectedTestResultModelSuccess,
+  UpdateTestResult
 } from './VehicleTestResultModel.actions';
 import { TestResultModel } from '@app/models/test-result.model';
+import { VIEW_STATE } from '@app/app.enums';
 
-const testResultModelExample: TestResultModel = {
-  testerStaffId: '21',
-  testStartTimestamp: new Date(),
-  odometerReadingUnits: 'test',
-  testEndTimestamp: new Date(),
-  testStatus: 'new',
-  testTypes: [],
-  vehicleClass: null,
-  testStationName: 'test',
-  vehicleId: '213',
-  noOfAxles: 2,
-  vehicleType: 'none',
-  countryOfRegistration: 'Ro',
-  preparerId: '231',
-  odometerReading: 123,
-  vehicleConfiguration: 'none',
-  testStationType: 'none',
-  reasonForCancellation: 'none',
-  testerName: '213',
+const testResultModelExample = {
```

Comment:
I would rename that expectedTestResultModel for better clarity in the unit test.
We are asserting on that so it would read more natural expect(myActions(..)).toBe(expectedTestResultModel)
I think it is nice if we keep the variables names descriptive and the follow pattern for unit testing
// arrange (prepare my unit test logic with the relevant variable names/results to be asserted, created, etc..)
// act (call my thing to have the logic evaluated)
// assert (check the logic needed and result)